apps/api/src/app/comparison/services/comparison.service.ts

```diff
@@ -364,11 +375,26 @@ export class ComparisonService {
         diffImageKey,
         misMatchPercentage,
         isSameDimensions,
-        state,
+        state
       }
     });
 
     console.log('Update image data');
     return from(testSession.then());
   }
+
+  private applyIgnoreAreas(image: PNG, ignoreAreas: IgnoreAreaType[]): Buffer {
```

Comment:
how fast is this method?

Reply:
not sure that I'm able to answer that question directly 
as `pixelmatch` doesn't have this functionality they propose this as workaround https://github.com/mapbox/pixelmatch/issues/9#issuecomment-499665722
it completely depends on image size 
do you know how we could test it?